### PR TITLE
hack,README: add quickstart-container script

### PR DESCRIPTION
**Pull request by steveej**
@riuvshin PTAL

### CODE REVIEW

**Comment by steveej** [comment]
I reduced the scope of this PR to only add the script and mention it in the README. Extending the tests will go into another PR.

**Comment by steveej** [comment]
@crawford PTAL

File: README.md [hunk 1]
```diff
@@ -4,8 +4,24 @@ Cincinnati is an update protocol designed to facilitate automatic updates. It de
 
 ## Quick Start
 
+Prepare custom environment variables
+
+```console
+# Please change these accordingly
+CINCINNATI_REGISTRY="https://quay.io"
+CINCINNATI_REPO="steveej/cincinnati-test-public"
```
Comment (crawford) [inline README.md]: I don't want to point people at a personal repo.

File: hack/quickstart-container.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    pkill "docker logs -f ${CINCINNATI_GB}"
+}
+trap cleanup EXIT
+
+# Run the graph builder
+echo Spawning graph-builder container...
+docker run -d --rm \
+  --label="${CINCINNATI_LABEL}" \
+  --name "${CINCINNATI_GB}" \
+  --env "RUST_LOG=actix_web=error,dkregistry=trace" \
```
Comment (crawford) [inline hack/quickstart-container.sh]: trace seems aggressive.

**Review by crawford — COMMENTED** [review]
Who is the intended consumer of this script?

**Comment by steveej** [comment]
> Who is the intended consumer of this script?

I see it as a example towards we can point for a hands-on demonstration of how a cincinnati stack is set up. It's also useful for quick integration tests.

File: hack/quickstart-container.sh (hunk 2 above)
Comment (steveej) [reply hack/quickstart-container.sh]: This is meant for demo purposes as it shows nicely what requests are being made. I'm fine to change that to whatever you suggest

Comment (steveej) [reply hack/quickstart-container.sh]: I lowered it to error too

File: README.md (hunk 1 above)
Comment (steveej) [reply README.md]: I was able to set up the test repositories under the redhat organization :raised_hands:

**Comment by steveej** [comment]
@lucab what's your opinion on this? Do you see this being helpful?

**Comment by lucab** [comment]
@steveeJ sorry, I missed your earlier ping on this. For demo purpose, I think it makes sense to keep this at the documentation level. It is more helpful for people to see a quick example than having to dig through the variables in a bash script. My preference would be for something short and clean like https://github.com/coreos/matchbox/blob/master/Documentation/deployment.md#docker.

**Comment by lucab** [comment]
> I'd prefer to have this script in the repository, even without a reference from the README. How about that?

I am not in strong opposition here, so let's go for that. Please keep the other change in README (variables for test repo), as I think it's valuable there.

**Comment by steveej** [comment]
> I am not in strong opposition here, so let's go for that. Please keep the other change in README (variables for test repo), as I think it's valuable there.

ACK and done.

**Pull request merged**